.github/docker-images/crossbuild_libcrypto_1.1.1.sh

```diff
@@ -44,6 +44,10 @@ case $os-$arch in
         openssl_target=linux-x86
         dockcross_target=manylinux2014-i686
         ;;
+    manylinux2014-aarch64)
+        openssl_target=linux-aarch64
```

Comment:
is linux-armv7 the 32 bit version of this?

Reply:
Yes.